pages/common/git-undo.md

```diff
@@ -0,0 +1,13 @@
+# git undo
+
+> Undo recent commits.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-undo>.
+
+- Remove most recent commit:
+
+`git undo`
+
+- Remove a specific number of recent commits:
```

Comment:
```suggestion
- Remove a specific number of most recent commits:
```

Reply:
@bl-ue opinion?

Reply:
Yep, I like that.  I'd add `the` before `most` in this and the other example, though.